@open-wc/testing-helpers oneEvent TS errors after upgrade to 4.0.0

## Actual Behavior

When upgrading to @open-wc/testing 4.0.0 we encounter following TypeScript errors in our tests:

```
error TS2554: Expected 3 arguments, but got 2.

        const event = oneEvent(fooFixture, 'barEventType') as Promise<BarEvent>;
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  node_modules/@open-wc/testing-helpers/types/src/helpers.d.ts:85:112
    85 export function oneEvent<TEvent extends Event = CustomEvent<any>>(eventTarget: EventTarget, eventName: string, preventDefault: boolean): Promise<TEvent>;
                                                                                                                      ~~~~~~~~~~~~~~~~~~~~~~~
    An argument for 'preventDefault' was not provided.
```

I guess in [types.d.ts](https://github.com/open-wc/open-wc/blob/9767b1ae8a52424e224d238784e04a9710d4e9b3/packages/testing-helpers/src/types.d.ts#L2) it should rather be:

```
export type OneEventFn =
  <TEvent extends Event = CustomEvent>(eventTarget: EventTarget, eventName: string)=> Promise<TEvent>
```
(so no `preventDefault` argument)

given the functions in [helpers.js](https://github.com/open-wc/open-wc/blob/9767b1ae8a52424e224d238784e04a9710d4e9b3/packages/testing-helpers/src/helpers.js#L126) look like:

`export function oneEvent(eventTarget, eventName)`

and 

`export function oneDefaultPreventedEvent(eventTarget, eventName)`

## Additional context

We are using TypeScript 5.2.2 and Node 21.0.0

If someone confirms, we might open a PR if desired

--- a/.changeset/moody-panthers-visit.md
+++ b/.changeset/moody-panthers-visit.md
@@ -1,0 +1,7 @@
+---
+'@open-wc/testing-helpers': patch
+---
+
+fix(testing-helpers): correct type declaration for `oneEvent`
+
+`preventDefault` parameter is not used

--- a/packages/testing-helpers/src/types.d.ts
+++ b/packages/testing-helpers/src/types.d.ts
@@ -1,3 +1,3 @@
 
 export type OneEventFn =
-  <TEvent extends Event = CustomEvent>(eventTarget: EventTarget, eventName: string, preventDefault: boolean)=> Promise<TEvent>
+  <TEvent extends Event = CustomEvent>(eventTarget: EventTarget, eventName: string) => Promise<TEvent>

